sources/packages/backend/apps/api/src/services/education-program-offering/education-program-offering.service.ts

```diff
@@ -719,6 +730,12 @@ export class EducationProgramOfferingService extends RecordDataModelService<Educ
       });
     }
 
+    if (options?.locationId) {
+      offeringQuery.andWhere("institutionLocation.id = :locationId", {
+        locationId: options?.locationId,
```

Comment:
may be we don't need null check inside the if

Reply:
Changed.